### PR TITLE
Feature/reactotron

### PR DESCRIPTION
**Pull request by ghost**
## Summary

[Change!] Install and configure Reactotron

### Android

image

### iOS

image

## Trello Card

[Change!] [Trello Card](https://trello.com/c/lUWh1J5h/26-n-57-debugging-ii-reactotron)

### CODE REVIEW

File: wbooks/index.js [hunk 1]
```diff
@@ -7,4 +7,8 @@ import { AppRegistry } from 'react-native';
 import App from './App';
 import { name as appName } from './app.json';
 
+if (__DEV__) {
+  import('./src/config/reactotronConfig');
+}
+
```
Comment (sfernandez11) [inline wbooks/index.js]: 2 things here:
* Move this to `App.tsx`:
```
import './src/config/reactotronConfig';
import App from './src/app';
...
```
* It's not necessary the `if (__DEV__)` for the import, just in the `reactotronConfig`  file is OK

File: wbooks/src/config/reactotronConfig.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  interface Console {
+    tron: Tron;
+  }
+}
+
+if (__DEV__) {
+  const { scriptURL } = NativeModules.SourceCode;
+  const scriptHostName = scriptURL.split('://')[1].split(':')[0];
+
+  Reactotron.configure({ name: 'appName', host: scriptHostName }).setAsyncStorageHandler!(
```
Comment (sfernandez11) [inline wbooks/src/config/reactotronConfig.ts]: `'appName'` is the correct name of this app?

**Review by sfernandez11 — CHANGES_REQUESTED** [review]
Great work! Justo two comments...

File: wbooks/index.js (hunk 1 above)
Comment (ghost) [reply wbooks/index.js]: Done!

File: wbooks/src/config/reactotronConfig.ts (hunk 2 above)
Comment (ghost) [reply wbooks/src/config/reactotronConfig.ts]: Hahah mi fault, I changed that..
Done!

**Review by sfernandez11 — APPROVED** [review]
🥇

**Pull request merged**